Remaining section review (#11)
ezsystems/commerce-documentation: docs/api/fields_for_ecommerce_data/fields_for_ecommerce_data.md

| Type             | Used for                                                     | identifier     |
| ---------------- | ------------------------------------------------------------ | -------------- |
| `TextLineField`  | A simple string without HTML code                            | `sestextline`  |
| `TextBlockField` | A rich text field containing HTML code                       | `sestextblock` |
| `ImageField`     | An image containing a path to an image and an alternative text | `sesimage`     |
| `ArrayField`     | A structured array                                           | `sesarray`     |
| `PriceField`     | An instance of the Price class                               | `sesprice`     |

## Templates

> For each concrete Field a template has to be provided to render the Field in the template. 

Reviewer: ```suggestion
For each concrete Field you have to provide a template to render the Field in the template. 
```

> The templates have to be provided in the `FieldTypes` folder. The name of the template has to start with the identifier of the Field, e.g.:

Reviewer: ```suggestion
The templates have to be put in the `FieldTypes` folder. The name of the template has to start with the identifier of the Field, e.g.:
```
